2215.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param {number[]} nums1
+ * @param {number[]} nums2
+ * @return {number[][]}
+ */
+var findDifference = function (nums1, nums2) {
+  const ans = [];
+  const ans0 = [];
+  const ans1 = [];
+
+  for (let i = 0; i < nums1.length; i++) {
+    if (!nums2.includes(nums1[i])) {
+      if (!ans0.includes(nums1[i])) {
+        ans0.push(nums1[i]);
+      }
+    }
```

Comment:
It's one of the best practices that you should avoid too much nesting because it would make it harder for others to read your code.